app/src/main/res/values-cy/strings.xml

```diff
@@ -176,91 +468,272 @@
+    -->
+    <string name="notification_browsing_session_channel_description">Mae hysbysiadau\'n caniatáu i chi ddileu eich sesiwn %1$s gyda thap. Does dim angen agor yr ap na gweld beth sy\'n rhedeg yn eich porwr.</string>
+
+    <!-- Label for a button in the "tabs tray" to erase the browsing history (closes all tabs and removes associated data). -->
+    <string name="tabs_tray_action_erase">Dileu hanes pori</string>
+
+    <!-- If the user decides to open the current page in another browser and if Firefox is not installed then we show this
+      item in the list of browsers. Clicking it will open the app store installed on the device (usually that's Google Play). -->
+    <string name="download_firefox">Llwytho Firefox i Lawr</string>
+
+    <!-- Text shown in the "Your rights" screen.
+    %1$s will be replaced with the name of the app (e.g. Firefox Focus)-->
+    <string name="your_rights_content1"><![CDATA[Mae %1$s yn feddalwedd cod agored a rhydd gan Mozilla a chyfranwyr eraill.]]></string>
+
+
+
+
+  <!-- Text shown in the "Your rights" screen.
+    %1$s will be replaced with the name of the app (e.g. Firefox Focus)
+    %2$s will be replaced with a URL linking to the MPL (e.g. https://www.mozilla.org/en-US/MPL/) -->
+    <string name="your_rights_content2"><![CDATA[Mae %1$s ar gael i chi o dan amodau <a href="%2$s">Trwydded Gyhoeddus Mozilla</a> a thrwyddedau cod agored eraill.]]></string>
+
+    <!-- Text shown in the "Your rights" screen.
+    %1$s will be replaced with the name of the app (e.g. Firefox Focus)
+    %2$s will be replaced with a URL linking to the trademark policy (e.g. https://www.mozilla.org/foundation/trademarks/policy/) -->
+    <string name="your_rights_content3"><![CDATA[Nid ydych yn derbyn unrhyw hawliau na thrwyddedau i nod masnach y Mozilla Foundation neu unrhyw barti arall, gan gynnwys enwau a logos Mozilla, Firefox n %1$s. Mae gwybodaeth bellach i’w gael <a href="%2$s">yma</a>.]]></string>
+
+    <!-- Text shown in the "Your rights" screen.
+    %1$s will be replaced with the name of the app (e.g. Firefox Focus)
+    %2$s will be replaced with a URL linking to the list of licenses used by the dependencies.-->
+    <string name="your_rights_content4"><![CDATA[Mae cod ffynhonnell ychwanegol ar gyfer %1$s ar gael o dan amrywiol <a href="%2$s">drwyddedau</a> rhydd a chod agored.]]></string>
+
+    <!-- Text shown in the "Your rights" screen.
+    %1$s will be replaced with the name of the app (e.g. Firefox Focus)
+    %2$s will be replaced with a URL linking to the GPL
+    %3$s will be replaced with a URL linking to the tracking protection wiki page (e.g. https://wiki.mozilla.org/Security/Tracking_protection#Lists) -->
+    <string name="your_rights_content5"><![CDATA[Mae %1$s hefyd yn defnyddio rhestrau rhwystro sy’n cael eu darparu gan Disconnect, Inc. fel gweithiau ar wahân ac annibynnol o dan <a href="%2$s">Trwydded Gyhoeddus Gyffredinol GNU v3</a>, ac sydd ar gael <a href = "% 3 $ s ">yma</a>.]]></string>
```

Comment:
```suggestion
    <string name="your_rights_content5"><![CDATA[Mae %1$s hefyd yn defnyddio rhestrau rhwystro sy’n cael eu darparu gan Disconnect, Inc. fel gweithiau ar wahân ac annibynnol o dan <a href="%2$s">Trwydded Gyhoeddus Gyffredinol GNU v3</a>, ac sydd ar gael <a href = "%3$s ">yma</a>.]]></string>
```

Nice catch Mihai!

cc: @Delphine if you could correct this on Pontoon.

Reply:
Corrected this, just needs to be approved @Delphine: https://pontoon.mozilla.org/cy/focus-for-android/mozilla-mobile/focus-android/app/src/main/res/values/strings.xml/?search=your_rights_content5&string=223986

Reply:
Updated in Pontoon! Just had to update the string one last time (also saw the localizer in the meantime had reverted your change, so had to explain to him why we did that).
Should be fixed now, thanks!